tests/trees/test_isoup_tree.py

```diff
@@ -80,7 +81,8 @@ def test_isoup_tree_perceptron(test_path):
     expected_predictions = np.load(test_file)
     assert np.allclose(y_pred, expected_predictions)
     error = mean_absolute_error(y_true, y_pred)
-    expected_error = 144.44877909957646
+
```

Comment:
Same comment about clarifying changes in model performance.

Reply:
Same situation. Unfortunately the problem encompassed both the single-target and multi-target regressors.